lib/users.js

```diff
@@ -38,35 +38,22 @@ users.sendPasswordResetEmail = async (userAccount) => {
 };
 
 users.signUp = async (userInfo) => {
-  try {
```

Comment:
I took out `try ... catch` here because the whole `signUp` function goes in another `try ... catch` in `userRoutes` function